packages/web-react/src/hooks/styleProps.ts

```diff
@@ -1,13 +1,70 @@
-import { CSSProperties, HTMLAttributes } from 'react';
-import { StyleProps } from '../types/shared/style';
+import { CSSProperties, HTMLAttributes, useContext } from 'react';
+import classNames from 'classnames';
+import { SpacingStyleProp } from '../constants';
+import ClassNamePrefixContext from '../context/ClassNamePrefixContext';
+import { BreakpointToken, SpaceToken, StyleProps } from '../types';
+import { applyClassNamePrefix } from '../utils';
 
 export type StylePropsResult = {
   styleProps: HTMLAttributes<HTMLElement>;
   props: HTMLAttributes<HTMLElement>;
 };
 
+export type StyleUtilitiesResult = {
+  styleUtilities: string[];
+  props: StyleProps;
+};
+
+const normalizeSpacingValue = (value: string): string => {
+  return value === 'auto' ? 'auto' : value.replace(/[^0-9]/g, '');
```

Comment:
There is an `auto` in multiple places. Constant instead?

Reply:
It is in the type definition and in this function too. Where should the constant be placed?